<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><!-- InstanceBegin template="/Templates/research2.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
<!-- InstanceBeginEditable name="doctitle" -->
<title>MIT UROP: Current Research - Biology: Course 7</title>
<!-- InstanceEndEditable -->
<link rel="stylesheet" href="../../urop_lev2.css" type="text/css"/>
<script type="text/javascript">
<!--
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
</head>
<body onload="MM_preloadImages('../../images/basicinfo_over.gif','../../images/forstudents_over.gif','../../images/forsupdept_over.gif','../../images/resources_over.gif','../../images/giving_over.gif')">
<div id="wrap">
<!-- begin main page header -->
<div id="mainheader"><a href="../../index.html"><img id="urop_lev1_rev_01" src="../../images/research_04.gif" width="769" height="97" border="0" alt="MIT's Undergraduate Research Opportunities Program (UROP)" /></a></div>
<!-- begin search code -->
<div id="search">
	<form method='get' action='http://search.mit.edu/search'>
				<label for ="searchbox">Search</label><input type="text" id="searchbox" name="q" value="" class="searchBox"/>
				<input name="btnG" type="submit" class="custombutton" value="Go"/>
				<input type='hidden' name='site' value='mit'/>
				<input type='hidden' name='client' value='mit'/>
				<input type='hidden' name='proxystylesheet' value='mit'/>
				<input type='hidden' name='output' value='xml_no_dtd'/>	
                <input type="hidden" name="ie" value="UTF-8"/> 
				<input type='hidden' name='as_dt' value='i'/>
				<input type='hidden' name='as_sitesearch' value="web.mit.edu/urop"/>
	</form>
<!-- end search code --></div>
<!-- begin main navigation -->
<div id="mainnavigation">
 <div id="lev2-13_"><img src="../../images/research_14.gif"  alt="" width="323" height="64" id="lev2_13" /></div> <div id="lev2-06_"><a href="../../basicinfo/index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('basicinfo','','../../images/basicinfo_over.gif',1)"><img src="../../images/basicinfo.gif" alt="Basic Information" name="basicinfo" width="116" height="64" border="0" id="basicinfo" /></a></div> 
 <div id="lev2-07_"><a href="../../students/index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('students','','../../images/forstudents_over.gif',1)"><img src="../../images/forstudents.gif" alt="For Students" name="students" width="94" height="64" border="0" id="students" /></a></div><div id="lev2-08_"><a href="../../dept/index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('dept','','../../images/forsupdept_over.gif',1)"><img src="../../images/forsupdept.gif" alt="For Supervisors and Departments" name="dept" width="132" height="64" border="0" id="dept" /></a></div>
<div id="lev2-09_"><img src="../../images/research_over.gif" width="104" height="64" alt="Current Research" /></div>
<div id="lev2-10_"><a href="../../resources/index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('resources','','../../images/resources_over.gif',1)"><img src="../../images/resources.gif" alt="Resources" name="resources" width="113" height="64" border="0" id="resources" /></a></div>
<div id="lev2-11_"><a href="http://giving.mit.edu/priorities/campaign/commons/urop.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('giving','','../../images/giving_over.gif',1)"><img src="../../images/giving.gif" alt="Giving to MIT" name="giving" width="107" height="64" border="0" id="giving" /></a></div>
</div>
<!-- end main navigation -->
<div id="mainnavigation2">
<div id="lev2-15_">
  <h1>Current Research</h1></div>
</div>
  <!-- end main page header -->
  <!--begin section navigation -->
  <div id="sectionnavigation">
    <div id="sectionlinks"><!-- InstanceBeginEditable name="sectionnavtext" -->
      <ul>
        <li><a href="../openings.html">Project Openings</a></li>
        <li><a href="index.html"><strong>Research Profiles</strong></a>
          <ul>
            <li><a href="course1.html">Course 1</a></li>
            <li><a href="course2.html">Course 2</a></li>
            <li><a href="course3.html">Course 3</a></li>
            <li><a href="course4.html">Course 4</a></li>
            <li><a href="course5.html">Course 5</a></li>
            <li><a href="course6.html">Course 6</a></li>
            <li class="onpage">Course 7</li>
            <li><a href="course8.html">Course 8</a></li>
            <li><a href="course9.html">Course 9</a></li>
            <li><a href="course10.html">Course 10</a></li>
            <li><a href="course11.html">Course 11</a></li>
            <li><a href="course12.html">Course 12</a></li>
            <li><a href="course14.html">Course 14</a></li>
            <li> <a href="course15.html">Course 15</a></li>
            <li><a href="course16.html">Course 16</a></li>
            <li><a href="course17.html">Course 17</a></li>
            <li><a href="course18.html">Course 18</a></li>
            <li><a href="course20.html">Course 20</a></li>
            <li><a href="course21a.html">Course 21A</a></li>
            <li><a href="course21f.html">Course 21F</a></li>
            <li><a href="course21h.html">Course 21H</a></li>
            <li><a href="course21l.html">Course 21L</a></li>
            <li><a href="course21m.html">Course 21M </a></li>
            <li><a href="course21w.html">Course 21W</a></li>
            <li><a href="course22.html">Course 22 </a></li>
            <li><a href="course24.html">Course 24</a></li>
            <li><a href="cms.html">Course 21CMS</a></li>
            <li><a href="esd.html">ESD</a></li>
            <li><a href="hst.html">HST</a></li>
            <li><a href="sts.html">STS</a></li>
            <li><a href="wgs.html">WGS</a></li>
            <li><a href="index.html#labs_ctrs">Lab &amp; Centers</a></li>
            <li><a href="index.html#other">Other Programs</a></li>
          </ul>
        </li>
        <li><a href="../other.html">Other Research</a></li>
      </ul>
    <!-- InstanceEndEditable --> </div>
  </div>
  <!--end sectionnavigation  -->
  <!-- begin main content -->
  <!-- InstanceBeginEditable name="maintext" -->
  <div class="maincontentsection" id="content">
    <h2><a href="http://web.mit.edu/biology/www/">Biology: Course 7 </a></h2>
    <p>The Biology Department provides opportunities for undergraduate research in the biology project laboratory courses and in the individual research laboratories throughout the Department. A number of techniques and approaches common to most of the research in the Department are taught in subject 7.02. For many projects, successful completion of 7.02 or equivalent experience in laboratory work is necessary. </p>
    <p>Credit for work in the research laboratories is obtained by registering for 7.UR (P/F) or 7.URG (letter grade). Before starting research, students must complete the required UROP applications.</p>
    <p>The listings provided here give a general description of the research work carried out in the laboratory of each faculty member. More detailed information can be obtained at <a href="http://web.mit.edu/biology/www/facultyareas/index.html">http://web.mit.edu/biology/www/facultyareas/index.html</a></p>
    <h3>Safety</h3>
    <p>All UROP Students must have a safety orientation by a laboratory safety representative <a href="http://web.mit.edu/biology/www/admin/operations/opsafety.html">(Lab Safety Officer</a>) prior to beginning work in the lab.</p>
    <h3>Some Related Areas for UROPs</h3>
    <p> <a href="whitehead.html">Whitehead Institute</a>, <a href="pilm.html">Picower Institute</a>, <a href="broad.html">Broad Institute</a>, <a href="course20.html">Biological Engineering </a>,and the <a href="hst.html">Harvard/MIT Division of Health Sciences and Technology</a>.</p>
    <p>For further information please contact the Biology Education Office, 68-120, <a href="mailto:undergradbio@mit.edu">undergradbio@mit.edu</a>, x3-4718.</p>
    <h3>Faculty Research Descriptions</h3>
    <dl>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/amon.html">Prof. Angelika Amon</a>,</strong> E17-233A, 8-8964, <a href="mailto:angelika@mit.edu">angelika@mit.edu</a></dt>
      <dd> Cell division is the fundamental process by which an organism is built. Deciphering the regulatory networks that govern cell division is therefore vital to understanding both normal cell division and the abnormal cell division that is a hallmark of cancer. We study the mechanisms that control the transitions from one cell-cycle stage to the next using budding yeast as a model system. We focus on how the anaphase - G1 transition, also known as exit from mitosis, is regulated and integrated with other cell cycle events and on how a specialized cell cycle, the meiotic cell cycle is established. These studies will hopefully shed light on the causes of meiotic chromosome mis-segregation, the leading cause of miscarriages in humans.</dd>
      <dt><strong></a><a href="http://mit.edu/biology/www/facultyareas/facresearch/baker.shtml">Prof. Tania A. Baker</a></strong>, 68-523, x3-3594, <a href="mailto:tabaker@mit.edu">tabaker@mit.edu</a></dt>
      <dd> Our goal is to understand the mechanism and regulation of two classes of macromolecular machines: the Clp/Hsp100 family of protein unfolding enzymes and the proteins that catalyze DNA transposition. These biological catalysts are being studied using biochemistry, structural biology, molecular biology, and genetics.</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/bartel.shtml">Prof. David P. Bartel</a>,</strong> Whitehead 601B, x8-5287, <a href="mailto:dbartel@wi.mit.edu">dbartel@wi.mit.edu</a></dt>
      <dd> We study RNA catalysts and small RNAs that regulate gene expression. With regard to regulatory RNAs, we are combining biochemical, molecular, and computational approaches to investigate post-transcriptional regulatory processes guided by microRNAs&mdash;short (~22-nt) RNAs found in plant and animal cells. This work is uncovering a widespread influence of microRNAs on metazoan gene expression and numerous specific cases in which microRNAs are playing crucial roles during plant and animal development. With regard to RNA catalysis, we use combinatorial methods to create new RNA enzymes (ribozymes). Learning the types of reactions that RNA can catalyze, how easy it is for new ribozymes to emerge, and other intrinsic properties of RNA provides a backdrop for understanding biocatalysis and is important for evaluating current notions of life's origins and early evolution</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/bell.html">Prof. Stephen P. Bell</a>,</strong> 68-630A, x3-2054, <a href="mailto:spbell@mit.edu">spbell@mit.edu</a></dt>
      <dd>We are using a combination of biochemistry, genetics and molecular biology to uncover the mechanisms responsible for the accurate and regulated duplication of eukaryotic chromosomes.</dd>
      <dt><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/boyer.html"><strong>Prof. Laurie Boyer</strong></a>, 68-230, x4-3335, <a href="mailto:lboyer@mit.edu">lboyer@mit.edu</a></dt>
      <dd>Stem cells are essential for metazoan development and for the maintenance of tissue homeostasis in the adult organism. Embryonic stem (ES) cells can be derived from the mammalian pre-implantation embryo and have enormous therapeutic potential because they can propagated in vitro while maintaining the capacity to give rise to all cell types in the body. A major challenge in biology is to understand how these undifferentiated cells execute the diverse gene expression programs that lead to cellular specification. Chromatin organization is a fundamental mechanism used by all eukaryotes to compartmentalize the genome into functional domains in order to interpret the vast amount of genetic information encoded within the genome. The overall goal of the lab is to understand how chromatin structure influences gene expression programs and ultimately cell fate and how failure to establish proper chromatin states can contribute to disease. To address these questions, we use a combination of genomic, genetic, biochemical
        and cell biological tools to precisely characterize the factors involved in regulating chromatin structure, to determine how these factors are recruited to genomic sites, and to investigate how these different regulatory pathways cooperate to organize the genome. We are particularly interested in how specific chromosomal domains are assembled and propagated in ES cells, adult stem cells, and somatic cells. Discovering how gene expression programs are regulated is required to improve our understanding of development and disease, and for realizing the therapeutic potential of stem cells. </dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/burge.html">Prof. Chris Burge</a> ,</strong> 68-223, x8-5997, <a href="mailto:cburge@mit.edu">cburge@mit.edu</a>, <a href="http://genes.mit.edu/burgelab/">Burge Lab Home Page</a></dt>
      <dd>Computational Biology of Gene Expression. We study mechanisms of gene regulation that occur post-transcriptionally using a combination of experimental and computational methods. A major goal is to understand the RNA splicing code: how the precise locations of exons and splice sites are identified in primary transcripts, and how this code is altered in mammalian development and differentiation. We use in vivo screens and computational methods to identify short RNA sequences that enhance or silence splicing, and study the roles of these sequences in the control of splicing decisions genome-wide. We also study the roles that microRNAs play in gene regulation, and we are developing methods to reliably predict microRNA regulatory targets and to understand their functions in gene networks. Eventually, we hope to gain a better understanding of how regulation at different steps in the expression of genes is integrated to produce the appropriate spectrum and levels of mRNA and protein isoforms under particular
        conditions </dd>
      <dt><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/chang.html"><strong>Prof. Paul Chang</strong></a>, E18-270, 4-3879, <a href="mailto:pchang2@mit.edu">pchang2@mit.edu</a></dt>
      <dd>Our lab has two primary areas of interest: 1. Understanding the mechanism of poly(ADP-ribose) function in cells and organisms and 2. Identifying novel molecules required for mitotic spindle assembly</dd>
      <dt><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/cheeseman.html"><strong>Prof. Iain Cheeseman</strong></a>, WI-401B, 4-2503, <a href="mailto:icheese@mit.edu">icheese@mit.edu</a></dt>
      <dd>Chromosome segregation during mitosis requires a large proteinaceous structure termed the kinetochore to generate attachments between chromosomal DNA and spindle microtubule polymers. The kinetochore is composed of more than 80 different proteins which function together to direct kinetochore assembly, generate dynamic connections with spindle microtubules, and regulate chromosome segregation. Our lab is interested in understanding the molecular basis of kinetochore function in human cells. We use parallel biochemical and cell biological approaches to examine kinetochore composition, structure, organization, regulation, and how kinetochore proteins function to achieve proper chromosome segregation.</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/chen.shtml">Prof. Jianzhu Chen</a>,</strong> E17-128C, x3-6173, <a href="mailto:jchen@mit.edu">jchen@mit.edu</a>, <a href="http://web.mit.edu/ccr/labs/chenlab/">Chen Lab Home Page</a></dt>
      <dd> Our research interests are to understand cellular and molecular mechanisms underlying the development and function of the immune system. We specifically study i) control of antigen receptor gene assembly and dysregulation of the process in lymphoid tumorigenesis, and ii) cellular and molecular basis of immunological memory, focusing on CD8 T cell responses to virus infection in the lung and prostate cancer. We are developing novel prophylaxis and therapies against virus infection in the lung.</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/chisholm.html">Prof. Sallie Chisholm</a>,</strong> 48-419, x3-1771, <a href="mailto:chisholm@mit.edu">chisholm@mit.edu</a>,<a href="http://web.mit.edu/chisholm/www/">Chisholm Lab </a></dt>
      <dd> The general goal of the research in my lab is to advance our understanding of microbial ecology and evolution in the oceans. In recent years we have focused our attention on a single group, the cyanobacterium Prochlorococcus, which is the smallest and most abundant microbe in ocean ecosystems &mdash; sometimes accounting for half of the total chlorophyll. This "minimal phototroph" can convert CO2, sunlight, and inorganic nutrients into a living cell with as few as 1700 genes. We have been developing Prochlorococcus, and the phage that infect them, as a model system for understanding life processes across all scales of spatial and temporal organization, from the genome to the biosphere, and from daily to evolutionary time scales. In so doing, we hope to develop a unified understanding of this one small representative of the diversity of life. </dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/cpaton.shtml">Prof. Martha Costantine-Paton</a>,</strong> 46-4165, x8-6415, <a href="mailto:mcpaton@mit.edu">mcpaton@mit.edu</a></dt>
      <dd> Our work concentrates on the glutamatergic and GABAergic neurotransmitter systems in the midbrain optic lobes, (the superior colliculus of rodents and the optic tectum of amphibians). The normal development of the visual projections to these regions is well understood and they are readily accessible for assays and manipulations of synaptic and structural plasticity that would be difficult or impossible in other regions of the central nervous system. </dd>
      <dt><a href="http://web.mit.edu/chemistry/www/faculty/drennan.html"><strong>Prof. Catherine Drennan</strong></a>, 16-573, 3-5622, <a href="mailto:cdrennan@mit.edu">cdrennan@mit.edu</a></dt>
      <dd>The Drennan laboratory uses X-ray crystallography as the chief tool for investigating the structure and function of enzymes that are medically important or valuable in environmental remediation. We are particularly interested in metalloprotein biochemistry and in the role of conformational change in catalysis.</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/eisen.shtml">Prof. Herman Eisen</a>,</strong> E17-128B, x3-6406, <a href="mailto:hneisen@mit.edu">hneisen@mit.edu</a></dt>
      <dd> T lymphocytes make up about 2-3% all cells in mammals. Most of these cells express an ab heterodimeric antigen-specific receptor (the T cell receptor or TCR), and fall into 2 groups: CD4 ("helper") cells and CD8 cytoyoxic cells (cytotoxic T lymphocytes or CTL). We study CD8 cells. The antigens they recognize are complexes formed by short peptides (8-10 amino acids in length) in association with class I proteins encoded by major histocompatibility genes (peptide-MHC or pep-MHC complexes). </dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/fink.shtml">Prof. Gerald R. Fink</a>,</strong> Whitehead 561F, x8-5215, <a href="mailto:gfink@wi.mit.edu">gfink@wi.mit.edu</a>, <a href="http://www.wi.mit.edu/research/summaries/fink.html">Fink Lab Home Page</a></dt>
      <dd>The goal of our research is to understand the connection between the yeast to filament switch and fungal virulence. This switch is intimately linked to the molecules that encircle the fungal cell, &beta;-glucan and the mannoproteins (adhesins) that decorate the &beta;-glucan. The analysis of filamentation in the model system, Saccharomyces cerevisiae guides our studies in the less tractable pathogen, Candida albicans. The genomes of both fungi encode many mannoproteins that confer unique adherence properties. These adhesins are required for interactions of fungal cells with each other (flocculation and filamentation), with inert surfaces (agar and plastic) and with mammalian cells. These cell surface molecules are the antigens recognized by the phagocytic cells of the immune system. Fungi are able to vary these cell surface molecules. We use genetics, biochemistry and genomics to address questions such as: What mechanisms generate the diversity of cell surface molecules? What cell surface molecules are
        recognized by the cells of the immune system? How do macrophages and neutrophils distinguish between a pathogen, Candida albicans, and a non-pathogen, Saccharomyces cerevisiae. </dd>
      <dt><strong><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/gehring.html">Prof. Mary Gehring</a>, </strong>WI-561C, x4-0343, <a href="mailto:mgehring@wi.mit.edu">mgehring@wi.mit.edu</a></dt>
      <dd>Epigenetics refers to heritable information that influences genome function but is not encoded in the DNA sequence itself. Cytosine DNA methylation is an epigenetic mark essential for transposable element silencing, genome stability, and genomic imprinting in a diverse array of organisms. Perturbations of DNA methylation patterns can lead to changes in gene expression programs and result in severe developmental defects. However, in both flowering plants and mammals, programmed changes in DNA methylation patterns are essential for gamete specification and the development of viable offspring. We use genetic, genomic, and molecular biology approaches to study aspects of epigenomic reprogramming during plant reproduction in the model plant <em>Arabidopsis thaliana</em>. We focus on the interplay among repetitive sequences, DNA methylation, and chromatin structure in these dynamic processes.</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/gertler.shtml"> Prof. Frank Gertler</a>,</strong> E18-215B, 3-5511, <a href="mailto:fgertler@mit.edu">fgertler@mit.edu</a></dt>
      <dd>Precise control of cell motility is essential for embryonic development and a wide variety of physiological and pathophysiological processes. Developmental defects, metastatic cancer and other diseases can result when regulation of cell movement is perturbed. I am interested in understanding how cell movement and changes in cell shape are controlled. Directed cell migration requires dynamic remodeling of the cytoskeleton in response to diverse arrays of diffusible and surface-bound extracellular signals. We would like to understand how cells transduce environmental signals into the mechanical forces necessary to drive directed movement. My research program combines mouse genetics, cell biological and biochemical approaches to investigate the interplay between signal transduction pathways and the actin cytoskeleton, and to deduce the functional importance of these regulatory systems in organismal development and disease etiology. One focus of the lab involves the study of cell motility and the control
        of cellular protrusions. A related second focus involves studying migration of neurons and their growth cones, actin-rich structures that guide developing axons and dendrites to their targets. We utilize fluorescence and time-lapse video microscopy of living cells and high-resolution electron microscopy to analyze and quantify these processes. </dd>
      <dt><strong><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/gilbert.html">Wendy Gilbert</a>, </strong>68-333, x3-3706, <a href="mailto:wgilbert@mit.edu">wgilbert@mit.edu</a></dt>
      <dd>The proteins of a cell are the primary determinants of cellular form and function. Regulation of the proteome is therefore the ultimate goal of signaling pathways that connect cell physiology to internal and external environmental cues. We study the molecular mechanisms and physiological functions of translational control of gene expression using genome-wide translation state profiling, molecular genetics, and biochemistry. <strong><a href="http://web.mit.edu/biology/www/Ar/99grossman.html"></dd>
      <dt><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/grossman.html"><strong>Prof. Alan D. Grossman</strong></a>, 68-530, x3-1515, <a href="mailto:adg@mit.edu">adg@mit.edu</a></dt>
      <dd> We use a combination of genetic, molecular, physiological, biochemical, cell-biological, and genomic approaches to study growth and development of <em>Bacillus subtilis</em>. Current work focuses on the cell cycle, chromosome organization, chromosome partitioning, control of gene expression, and cell cell signaling. In addition, we wish to understand the mechanisms used to control the initiation of sporulation and the regulatory response to high population density (the quorum response). This involves defining the external stimuli needed to initiate spore formation and the quorum response, identifying the cellular components involved in sensing and transducing these signals through the cell, and identifying and characterizing genes that are activated in response to these conditions.</dd>
      <dt><strong><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/guarente.shtml">Prof. Leonard Guarente</a>,</strong> 68-280A, x3-6965,<a href="mailto:leng@mit.edu"> leng@mit.edu</a></dt>
      <dd><a href="http://web.mit.edu/biology/guarente/">Guarente Lab Home Page</a></dd>
      <dd>Study of molecular mechanisms regulating aging. Use of model systems Saccharomyces cerevisiae, C. elegans, mice, mammalian cells, and relevance to aging in humans. Study mechanisms by which SIR2-related genes regulate life span and calorie restriction (CR). </dd>
      <dt><strong>Prof. Piyush Gupta,</strong> <a href="mailto:piyush@mit.edu">piyush@mit.edu</a></dt>
      <dd></dd>
      <dt><strong><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/hemann.html">Prof. Michael T. Hemann</a></strong>, E17-128, x4-1964, <a href="mailto:hemann@mit.edu">hemann@mit.edu</a></dt>
      <dd>Many human cancers fail to effectively respond to chemotherapy, and cancers that initially respond frequently acquire drug resistance and relapse. Our lab uses emerging RNAi screening technology combined with murine stem reconstitution and tumor transplantation systems to investigate the genetic basis for intrinsic and acquired chemotherapeutic resistance. Our aim is to use these tractable mouse models to identify novel cancer drug targets, as well as strategies for tailoring existing cancer therapies to target the vulnerabilities of specific malignancies. </dd>
      <dt><strong><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/hopkins.shtml">Prof. Nancy Hopkins</a>,</strong> E17-342, x3-6414, <a href="mailto:nhopkins@mit.edu">nhopkins@mit.edu</a></dt>
      <dd> Identification of genes essential for early development in zebrafish, and the role of these genes in longevity and cancer predisposition in adult fish. Large-scale forward genetic screens are a powerful approach to identifying the genetic basis of developmental processes. Such screens have long been central to studies of invertebrate animals, and more recently the approach has been applied to vertebrate animals, including zebrafish and mice. Forward screens are particularly suitable in the fish. This is because it is possible to breed and maintain large numbers of zebrafish in the lab, and because early developmental mutations are easy to identify in fish embryos since embryos develop outside the mother and are transparent for the first week of life.</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/horvitz.shtml">Prof. Robert Horvitz</a></strong>, 68-425A, x3-4671, <a href="mailto:horvitz@mit.edu">horvitz@mit.edu</a><a href="http://web.mit.edu/horvitz/www/">Horvitz Lab Home Page</a>, <a href="http://www.hhmi.org/research/investigators/horvitz.html">Horvitz Page at HHMI</a></dt>
      <dd>Molecular, developmental and behavioral genetics of the nematode Caenorhabditis elegans. Use of genetics, molecular biology, cell biology, biochemistry and electrophysiology to analyze C. elegans development and behavior. Human molecular genetics, with a focus on the neurodegenerative disease amyotrophic lateral sclerosis.</dd>
      <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/housman.html">Prof. David Housman</a>,</strong> E17-521, x3-3013, <a  href="mailto:dhousman@mit.edu">dhousman@mit.edu</a></dt>
      <dd><a href="http://web.mit.edu/hman-genomics/www/">Housman Lab Home Page</a>.</dd>
      <dd>The research goals of my laboratory group involve the utilization of genetic approaches to the identification of mechanistic bases of human disease pathology and the utilization of this knowledge to develop effective strategies for intervention in human disease. Our efforts are focused in three major disease areas: trinucleotide repeat disorders particularly Huntington's disease (HD), cancer and cardiovascular disease.</dd>
      <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/hynes.shtml">Prof. Richard Hynes</a>,</strong> E17-227, x3-6422, <a  href="mailto:vingram@mit.edu">rohynes@mit.edu</a></dt>
      <dd><a href="http://web.mit.edu/ccrhq/hyneslab/">Hynes Lab Home Page</a>.</dd>
      <dd>The Hynes lab is interested in understanding the molecular basis of cell adhesion and its involvement in cell behavior including contributions to various human diseases, especially cancer progression, including invasion, metastasis and angiogenesis. Around 5-10% of the genes in a mammalian genome are involved in cell adhesion. The lab uses genetically engineered mice and cells derived from them, combined with molecular and cell biological methods, to investigate the roles of adhesion molecules in both normal physiology and in mouse models of human diseases.</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/imperiali.shtml">Prof. Barbara Imperiali</a>, </strong>18-590, x3-1838, <a href="mailto:imper@mit.edu">imper@mit.edu</a>,<a href="http://web.mit.edu/imperiali/">Imperiali Lab</a><a href="http://web.mit.edu/imperiali/"> </a></dt>
      <dd>Research in the Imperiali group is concerned with diverse aspects of protein structure, function and design. One area of investigation focuses on understanding the molecular details of the enzyme-catalyzed process of asparagine-linked protein glycosylation. A second program focuses on the development of new chemical and biochemical tools that can be implemented for the investigation of complex biochemical processes. Our approaches to these research areas at the interface between chemistry and biology are highly interdisciplinary and integrate organic synthesis, state-of-the-art spectroscopic analysis, enzymology, protein biochemistry and molecular biology.</dd>
      <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/jacks.shtml">Prof. Tyler Jacks</a>,</strong> E17-517A, x3-0262, <a  href="mailto:tjacks@mit.edu">tjacks@mit.edu</a>, <a href="http://www.hhmi.org/research/investigators/jacks.html">Jacks Page at HHMI</a></dt>
      <dd>My laboratory is interested in the genetic events that contribute to the development of cancer. The focus of our research is a series of mouse strains engineered to carry mutations in genes known to be involved in human cancer. Using conventional and conditional loss-of-function and gain-of-function mutations in various tumor suppressor genes as well as the K-ras oncogene, we have constructed mouse models of lung cancer, astrocytoma, endometroid ovarian cancer, retinoblastoma, and tumors of the peripheral nervous system. We also study the effects of these mutations on normal embryonic development and use cells derived from mutant animals to study the function of these genes in cell culture models.</dd>
      <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/jaenisch.shtml">Prof. Rudolf Jaenisch</a>, </strong>Whitehead 461B, x8-5186, <a href="mailto:jaenisch@wi.mit.edu">jaenisch@wi.mit.edu</a></dt>
      <dd> Our long range goals are to understand epigenetic regulation of gene expression in mammalian development and disease. Faulty epigenetic reprogramming is the main problem in the development of cloned mammals produced by nuclear transfer and understanding its molecular basis is a major focus of our work. DNA methylation is a crucial component of the epigenetic control of gene activity through the regulation of chromatin state. A number of factors, such as methyl-binding proteins and histone-deacetylases, have been identified that are involved in this process. We are deriving mice carrying targeted mutations in the different components of the epigenetic machinery to understand how stable expression states of the genome are established and maintained. In particular we are focusing on the role of DNA methylation in cancer, genomic imprinting and in the function of the postnatal brain and are employing gene-targeting methods to generate embryonic stem cells and mice with lineage-specific and inducible gene
        deletions. </dd>
      <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/kaiser.shtml">Prof. Chris Kaiser</a>,</strong> 68-533, x3-9804, <a  href="mailto:ckaiser@mit.edu">ckaiser@mit.edu</a></dt>
      <dd>We study fundamental mechanisms of protein folding and intracellular trafficking using the yeast <em>S. cerevisiae</em> as a model organism. Our work focuses on the folding of proteins in the endoplasmic reticulum (ER), quality control mechanisms in the ER, and membrane protein sorting in Golgi compartments. We use combined genetic, biochemical and cell biological methods to gain an understanding of the molecular mechanisms that underlie each of these processes</dd>
      <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/keating.html">Prof. Amy Keating</a></strong> , 68-622A, x2-3398, <a href="mailto:keating@mit.edu">keating@mit.edu</a></dt>
      <dd>Computational, biophysical, structural and proteomic studies of protein-protein interactions. In the Keating lab we are studying the versatility and specificity of protein-protein interactions through a combined program of bioinformatic analysis, structural modeling, computational design, and experimental characterization. Our aim is to understand, at a high level of detail, specific protein-protein interactions that establish normal or aberrant protein functions or pathways in the cell. We focus on simple protein domains and motifs, such as the a-helical coiled coil. Our research merges sequence-based and data-based descriptions of the proteome with physics-based descriptions of protein structure.</dd>
  &gt;
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/kim.html">Prof. Dennis H. Kim</a></strong>, 68-430A, x4-0050, <a href="mailto:dhkim@mit.edu">dhkim@mit.edu</a></dt>
  <dd>The primary research interest of my laboratory is to understand the common mechanisms that phylogenetically diverse organisms employ to defend themselves against microbial pathogens. Our experimental focus is the immune system of the nematode Caenorhabditis elegans. We take a genetic approach to identify determinants of immunity in the worm, making use of recent technological advances such as RNA interference that facilitate the genome-wide characterization of gene function. We anticipate that the investigation of pathogen resistance in C. elegans will provide important insights into evolutionarily conserved host organism responses to pathogen infection, with implications for the understanding of the evolution and function of vertebrate innate immunity.</dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/king.shtml">Prof. Jonathan King</a>,</strong> 68-330, x3-4700, <a  href="mailto:jaking@mit.edu">jaking@mit.edu</a>, <a href="http://web.mit.edu/king-lab/www/index.html">King Lab Home Page</a></dt>
  <dd>Deciphering the rules through which the amino acid sequences of polypeptide chains direct the folding and assembly of &szlig;-sheet proteins: Identification and characterization of the misfolded and inclusion body states that represent the failure of protein folding processes, with particular emphasis on folding defects from thermal stress. Investigation of the folding and association of human lens crystallins implicated in cataract formation. Protein folding and thermal stress responses in marine cyanobacteria and their phages: protein visualization tools for instruction and science education.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/krieger.html">Prof. Monty Krieger</a>,</strong> 68-483, x3-6793, <a  href="mailto:krieger@mit.edu">krieger@mit.edu</a></dt>
  <dd>Cell and molecular biology, lipoprotein receptors, lipoprotein and cholesterol metabolism, intracellular protein sorting, golgi function, somatic cell genetics, atherosclerosis, scavenger receptors, pathogen receptors, macrophage physiology, pattern recognition in vertebrate and invertebrate immune systems.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/lander.shtml">Prof. Eric Lander</a>, </strong>7 Cambridge Center, x2-1906, <a href="mailto:lander@broad.wi.mit.edu">lander@broad.wi.mit.edu</a>,<a href="http://www.broad.mit.edu/about/bios/bio-lander.html">Lander Lab </a></dt>
  <dd>With the successful completion of the Human Genome Project, the challenge now is to decipher the information encoded within the human genetic code - including genes, regulatory controls and cellular circuitry. Understanding these components, controls and circuits is fundamental to the study of physiology in both health and disease. The Broad Institute brings together a community focused on the comprehensive understanding of genomes through genome comparison to reveal functional elements through evolutionary conservation, studies of regulatory control by proteins and chromatin structure, and characterization of cell circuitry through monitoring and modulation of cellular states. The Institute is home to the research laboratories that were previously known as the Whitehead Institute/MIT Center for Genome Research (WICGR). Founded in 1990, the WICGR served as a flagship for the international collaborations to produce a draft sequence of the human and mouse genomes.</dd>
  <dt><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/laub.html"><strong>Prof. Michael Laub</strong></a>, 68-623, x4-0418, <a href="mailto:laub@mit.edu">laub@mit.edu</a>, <a href="http://laublab.mit.edu/">Laub Lab</a></dt>
  <dd>Our lab studies the regulation of cell cycle progression in bacteria using a combination of genetics, biochemistry, and cell biology. We are currently interested in systematically identifying the key transcriptional factors that control progression through the Caulobacter crescentus cell cycle. To this end, we are using newly developed techniques to rapidly delete and characterize every transcription factor encoded in the Caulobacter genome. We are looking for an enthusiastic, hard working student to participate in this project. Biology coursework, including 7.02, is required. Prior laboratory experience in molecular biology is recommended, but not required. </dd>
  <dt> <strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/lauffenburger.shtml">Prof. Douglas Lauffenburger</a>, </strong>56-341, x2-1629, <a href="mailto:lauffen@mit.edu">lauffen@mit.edu</a></dt>
  <dd><a href="http://web.mit.edu/dallab/">Lauffenburger Lab </a></dd>
  <dd>It is becoming increasingly apparent that to more fully understand how cells operate as integrated molecular systems, an intimate combination of quantitative experiment and computational modeling is required. This is clearly the case for the complex receptor-mediated signaling networks activated by growth factors, cytokines, and extracellular matrix. Accordingly, we are attempting to develop multi-variable, multi-parametric 'systems biology' measurement and analysis methodologies useful for investigating signaling networks and cell functions they regulate.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/lees.shtml">Prof. Jacqueline Lees</a>, </strong>E17-517B, x2-1972, <a  href="mailto:%20jalees@mit.edu">jalees@mit.edu</a></dt>
  <dd><a  href="mailto:%20jalees@mit.edu"></a>Our research is focused on identifying the proteins and pathways that play a key role in tumorigenicity and establishing the mechanism of their action in both normal and tumor cells. We approach this using a combination of molecular and cellular analyses, mutant mouse models and genetic screens in zebrafish.</dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/lindquist.html">Prof. Susan Lindquist</a>,</strong> Whitehead 661, x8-5184, <a href="mailto:lindquist_admin@wi.mit.edu">lindquist_admin@wi.mit.edu</a></dt>
  <dd><a href="http://web.wi.mit.edu/lindquist/pub/">Lindquist Lab </a></dd>
  <dd>The central theme of our research is to explore the impact of protein conformational changes on diverse processes in cellular and organismal biology. We are exploiting our understanding of protein folding to gain insights into the basis of neurodegenerative diseases and spongiform encephalopathies, and to design therapeutic strategies. In addition to the role that misfolded proteins play in disease, we have also identified potentially important beneficial effects of self-perpetuating alternate protein conformations, including mechanisms of evolutionary change and long-term memory.</dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/littleton.html">Prof. J. Troy Littleton</a>,</strong> 46-3243, 452-2605, <a href="mailto:troy@mit.edu">troy@mit.edu</a>, <a href="http://web.mit.edu/flybrain/littletonlab/">Littleton Lab</a></dt>
  <dd><a href="http://web.mit.edu/flybrain/littletonlab/"></a>The computational power of the brain depends on synaptic connections that link together billions of neurons. The focus of my laboratory's work is to understand the mechanisms by which neurons form synaptic connections, how synapses transmit information, and how synapses change during learning and memory. To complement this basic research in neuroscience, we also study how alterations in neuronal signaling underlie several neurological diseases, including epilepsy and Huntington's Disease. We combine molecular biology, protein biochemistry, microarray technology, electrophysiology, and imaging approaches with Drosophila genetics to address these questions. Moving beyond genomic data to ultimately understand how proteins specify the distinctive signaling properties of neurons and enable them to interconnect into computational circuits that dictate behavior are major goals for the next decade. Despite the dramatic differences in complexity between
    Drosophila and humans, genomic analysis has confirmed that key neuronal proteins and the functional mechanisms they govern are remarkably similar. As such, we are attempting to elucidate the mechanisms underlying synapse formation, function and plasticity using Drosophila as a model system. By characterizing how neurons integrate synaptic signals and modulate synaptic growth and strength, we hope to bridge the gap between molecular components of the synapse and the physiological responses they mediate. To accomplish these goals, the lab has developed a research plan that encompasses a genetic and functional analysis of several aspects of synaptic biology.</dd>
  <dt> <strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/lodish.shtml">Prof. Harvey F. Lodish</a>, </strong>Whitehead 601C, x8-5216, <a  href="mailto:lodish@wi.mit.edu">lodish@wi.mit.edu</a></dt>
  <dd><a href="http://web.wi.mit.edu/lodish/">Lodish Lab </a></dd>
  <dd>Research in my lab focuses on five important areas at the interface between molecular cell biology and medicine: Red blood cell development, especially on the role of three signal transduction pathways downstream of the erythropoietin receptor in controlling terminal proliferation and differentiation of erythroid progenitor cells; Hematopoietic stem cells, defining new cell surface proteins for their purification and new growth factors that support their expansion in culture; MicroRNAs, defining their roles in lineage commitment of hematopoietic stem and progenitor cells, and regulating muscle differentiation; Adiponectin, a hormone we cloned that is made exclusively by fat cells and that increases fatty acid and glucose metabolism by muscle, and four homologous proteins; and Regulated cleavage and release of the extracellular domain (&quot;ectodomain shedding&quot;) of transmembrane precursors of several secreted growth factors. </dd>
  <dt> <strong><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/martin.html">Prof. Adam Martin</a>,</strong> 68-459, x4-0074, <a href="mailto:acmartin@mit.edu">acmartin@mit.edu</a></dt>
  <dd>During embryonic development, masses of cells undergo dramatic rearrangements to organize into separate layers that will give rise to different parts of the body. This incredibly dynamic process is called gastrulation and is driven by cell shape changes that collectively deform the tissue. Our lab is interested in imaging the dynamics of these cell shape changes and determining how mechanical forces are generated that drive massive tissue movements. We study these questions using the embryonic development of the fruit fly, <em>Drosophila melanogaster,</em> where cell shape changes and cytoskeletal dynamics can be readily imaged, quantified, and functionally dissected using a multidisciplinary approach. </dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/nedivi.shtml">Prof. Elly Nedivi</a>,</strong> E18-670, x3-2344, <a href="mailto:nedivi@mit.edu">nedivi@mit.edu</a></dt>
  <dd><a href="http://monster.mit.edu/nedivi-lab/">Nedivi Lab Home Page</a></dd>
  <dd>The Nedivi lab studies the cellular mechanisms that underlie activity-dependent plasticity in the developing and adult brain. Their approach is to identify and characterize participating genes and the functions of the proteins they encode. This work began with the cloning of a large number of candidate plasticity genes (CPGs) that are activity regulated. The first two of these CPGs selected for in-depth analysis were cpg2 and cpg15, based on their expression and regulation patterns and the sub-cellular localization of their gene products.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/orrweaver.html">Prof. Terry Orr-Weaver</a>, </strong>Whitehead 561B, x8-5245, <a  href="mailto:weaver@wi.mit.edu">weaver@wi.mit.edu</a>, <a href="http://web.wi.mit.edu/orr-weaver/home.html">Orr-Weaver Lab </a></dt>
  <dd>The regulation of metazoan DNA replication and chromosome segregation. The coordination of cell division and the cell cycle with development. Our research goal is to decipher how the two fundamental steps in cell division, DNA replication and chromosome segregation, are regulated during the development of multicellular organisms. This coordination not only involves controlling the number of cell division cycles, but also the implementation of modified cell cycles for particular developmental strategies. Haploid gametes are produced by meiosis, and in oogenesis the meiotic cell cycle is linked to oocyte differentiation by developmentally triggered arrest and release points. Organisms that undergo rapid embryogenesis utilize an abbreviated cell cycle without growth phases. A third variant cell cycle, the endo cycle, is employed in specific tissues in most plants and animals. In the endo cycle DNA replication occurs but not mitosis, producing large polyploid or polytene cells with high metabolic activity.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/page.shtml">Prof. David Page</a>, </strong>Whitehead 401, x8-5203, <a  href="mailto:page_admin@wi.mit.edu">page_admin@wi.mit.edu</a></dt>
  <dd><a href="http://www.hhmi.org/research/investigators/page.html">Page at HHMI</a></dd>
  <dd>We study mammalian germ cells and their mitotic development, with particular attention to the roles of sex-chromosomal genes. Some of our work focuses on men who are infertile because of genetic defects disrupting germ cell development. Parallel studies in mice employ a rich array of genetic and embryologic tools. We have completely sequenced the human Y chromosome and analyzed its gene content. Many Y-linked genes, and a surprising number of X-linked genes, are expressed only in male germ cells. An unexpected product of our research is a new understanding of the sex chromosomes' evolutionary origins and dynamics.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/pardue.shtml">Prof. Mary Lou Pardue</a>,</strong> 68-670, x3-6741, <a  href="mailto:mlpardue@mit.edu">mlpardue@mit.edu</a></dt>
  <dd>Genetic, biochemical, and cytological studies of structural elements of chromosomes, with emphasis on telomeres, heterochromatin, and transposable elements. Studies of the coordination of nuclear and cytoplasmic activities. Analysis of the molecular mechanisms by which cells respond to stress, especially the molecular biology of the heat shock response.</dd>
  <dt><a href="http://wi.mit.edu/research/faculty/ploegh.html"><strong>Prof Hidde Ploegh</strong></a>, Whitehead 361B, x4-1878, <a href="mailto:ploegh@wi.mit.edu">ploegh@wi.mit.edu</a></dt>
  <dd>Our bodies are in a constant state of war with countless microbes mounting attacks on us. The Ploegh lab researches the dynamics of this protracted chess game between our bodies and these outside invaders. We study several cellular processes involved in the normal function of the immune system. In particular, antigen processing and presentation in MHC molecules has been and continues to be a major focus of our research. In the past several years we have been studying viral proteins that interfere with these processes, attempting to define the molecular mechanism of the interference.<a href="http://www.wi.mit.edu/research/fellows/rubins.html">http://www.wi.mit.edu/research/fellows/rubins.html</a>"We study poxviruses (smallpox, monkeypox and vaccinia viruses) and filoviruses (Ebola and Marburg virus). We analyze the gene expression responses of both the virus and the host cell during viral infection, using whole-genome DNA microarrays. Approaches include in vivo models of poxvirus and filovirus infection,
    analysis of immune cell response to virus infection, and in vitro tissue culture models of virus-host cell interaction. In addition we have ongoing therapeutic and vaccine studies for treatment of these deadly diseases. None of the live virus work is done at MIT/Whitehead, all samples are inactivated and safe for work in a Biosafety Level 1 environment." </dd>
  <dt> <strong><a href="http://mit.edu/bcs/people/quinn.shtml">Prof. William Quinn</a></strong>, E25-436, x3-6307, <a  href="mailto:cquinn@wccf.mit.edu">cquinn@wccf.mit.edu</a></dt>
  <dd> Genetic and Molecular Studies of Learning and Memory in Drosophila: Fruit flies can learn. They can identify a specific chemical odor that they have experienced with electric shock and avoid it. Moreover, they can remember to avoid it for several days. The Quinn lab is investigating the molecular mechanisms underlying learning acquisition and memory storage by inducing and selecting single-gene mutations that affect learning or memory, and by engineering transgenic fly strains that disrupt these processes.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/rajbhandary.shtml">Prof. Uttam RajBhandary</a>,</strong> 68-671A, x3-4702, <a  href="mailto:bhandary@wccf.mit.edu">bhandary@wccf.mit.edu</a></dt>
  <dd> Transfer RNAs, Protein Synthesis, Suppressor tRNAs, RNA-protein Interactions and Proteins Carrying Unnatural Amino Acids: From assembly of RNA viruses to mRNA localization during development, RNA-protein interactions play a crucial role in gene expression, gene regulation and development. The many proteins with which tRNAs interact during protein synthesis make tRNAs excellent systems for investigating specific RNA-protein interactions. We study tRNA structure, function, and biosynthesis using biochemistry, genetics and in vivo functional studies. We also work on the use of suppressor tRNAs to generate proteins carrying one or more unnatural amino acids in bacteria and in mammalian cells.</dd>
  <dt><a href="http://mit.edu/biology/www/facultyareas/facresearch/reddien.html"><strong>Prof. Peter Reddien</strong></a>, Whitehead 501B, x4-4083, <a href="mailto:reddien@wi.mit.edu">reddien@wi.mit.edu</a></dt>
  <dd>Regeneration is one of the great mysteries of biology. Planarians are bilaterally symmetric metazoans that possess almost unlimited regenerative capacities and that have been a classic regeneration model for over a century. Since planarian regeneration involves a population of adult pluripotent stem cells (the neoblasts), planarians are an excellent organism for studies of in vivo stem cell regulation. We use RNA interference (RNAi) for high-throughput studies of gene function in the planarian S. mediterranea. Our aim is to understand how planarian neoblasts control regeneration. </dd>
  <dt><a href="http://www.broad.mit.edu/about/bios/bio-regev.html"><strong>Aviv Regev</strong></a>, NE30-6013, x4-4911, <a href="mailto:aregev@broad.mit.edu">aregev@broad.mit.edu</a></dt>
  <dd> Molecular networks are the information processing devices of cells and organisms, transforming extra- and intra-cellular signals into coherent cellular responses. Understanding the function and evolution of molecular networks is a fundamental question in biology and medicine. Genomics provides powerful tools with which to probe the components and behavior of molecular networks: sequencing complete genomes, measuring the expression of every gene, and charting the biochemical and genetic interactions in the cell. My group combines computation and experiments that leverage genomics to quantitatively understand molecular networks: discover their components and connections; decipher the way in which relevant information is encoded at different layers of the network and translated into cellular responses; determine how multiple networks are integrated together; and reconstruct how contemporary complex systems have evolved over time to achieve their specific organization and remarkable functionality.</dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/rich.shtml">Prof. Alexander Rich</a>, </strong>68-233, x3-4715, <a  href="mailto:cbeckman@mit.edu">cbeckman@mit.edu</a></dt>
  <dd>Alternative nucleic acid motifs. Z-DNA Studies. RNA editing. Z-DNA is an alternative left-handed conformation of the DNA double helix. It was first visualized 20 years ago in this laboratory in an X-ray diffraction analysis. The immediate question asked at the time was: Does it have a biological function? An important advance was made by publication of the crystal structure of Z-DNA bound to a protein domain of an editing enzyme. The crystal structure reveals a detailed fitting of the protein to the DNA, held together by a system of 11 hydrogen bonds and 5 distinct van der Waals interactions. The protein binds to 5 successive phosphate residues on one strand of the Z-DNA double helix and also has a van der Waals contact which can only occur if the base is in the syn conformation which is a hallmark of Z-DNA. This protein domain is one of a family of proteins in which the important residues involved in Z-DNA interaction are strongly conserved.</dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/sabatini.html">Prof. David Sabatini</a>,</strong> Whitehead 361, x8-6407, <a href="mailto:sabatini@wi.mit.edu">sabatini@wi.mit.edu</a></dt>
  <dd><a href="http://web.wi.mit.edu/sabatini/">Sabatini Lab</a></dd>
  <dd>Our lab is interested in the basic mechanisms that regulate growth, the process whereby cells and organisms accumulate mass and increase in size. The pathways that regulate growth are often deranged in human diseases, such as diabetes and cancer. Our long-term goals are to identify and characterize these mechanisms and to understand their roles in the normal and diseased physiology of mammals. Our current focus is on a cellular system called the Target of Rapamycin (TOR) pathway, a major regulator of growth in many eukaryotic species. In addition to our work on growth control, we are developing and applying new technologies that facilitate the analysis of gene function in mammalian cells. We have developed &lsquo;cell-based microarrays' that allow us to look at the cellular effects of perturbing the activity of thousands of genes in parallel. We are also a founding member of a consortium of labs in the Boston area that is developing and using a genome-scale RNA interference (RNAi) library targeting
    human and mouse genes.</dd>
  <dt><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/saeij.html"><strong>Prof. Jeroen Saeij</strong></a>, 68-270, 4-5330, <a href="mailto:jsaeij@mit.edu">jsaeij@mit.edu</a></dt>
  <dd>The primary research interest of our laboratory is to understand how intracellular parasites exploit and manipulate the host cells in which they live, to ensure their survival, replication, and transmission, and hence their success. Our experimental focus is Toxoplasma gondii, considered the most successful protozoan parasite of warm-blooded animals. We are interested in identifying Toxoplasma proteins involved in the modulation of the host cell and the exact mechanisms by which they act. To achieve this we use a combination of genomics, biochemistry, genetics, microscopy, immunology and computational tools. Thus, students who join the lab can expect to be trained in a wide variety of techniques and learn how to extract biological meaning out of large amounts of data.</dd>
  <dt><strong><a href="http://samsonlab.mit.edu/">Prof. Leona Samson</a>,</strong> 56-235, x8-7813,<a href="mailto:lsamson@mit.edu">lsamson@mit.edu</a></dt>
  <dd>Alkylating agents represent an abundant class of chemical DNA damaging agent in our environment and they are toxic, mutagenic, teratogenic and carcinogenic. Since we are continuously exposed to alkylating agents, and since certain alkylating agents are used for cancer chemotherapy, it is important to understand exactly how cells respond when exposed to these agents. The repair of DNA alkylation damage provides tremendous protection against the toxic effects of these agents and our aim is to understand the biology, the biochemistry, and the genetics of numerous DNA repair pathways that act upon DNA alkylation damage.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/sauer.shtml">Prof. Robert Sauer</a>,</strong> 68-571A, x3-3163, <a  href="mailto:bobsauer@mit.edu">bobsauer@mit.edu</a>, </dt>
  <dd> <a href="http://mit.edu/sauerlab">Sauer Lab Website</a></dd>
  <dd>Our lab studies macromolecular function and folding using biochemistry, molecular genetics, protein design, and structural biophysics. We also study mechanisms of co-translational tmRNA-mediated protein tagging, AAA+ protein unfolding and degradation machines, and protease systems that signal between cell compartments. </dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/schwartz.shtml">Prof. Thomas Schwartz</a></strong>, 68-480, x 2-3851, <a href="mailto:tus@mit.edu">tus@mit.edu</a></dt>
  <dd>Research in our lab aims at understanding protein function on the basis of atomic structure determination using x-ray crystallography as the main tool. One focal point is the elucidation of the structure of the nuclear pore complex, an elaborate macromolecular protein assembly that constitutes the only gateway into and out of the eukaryotic cell nucleus. A second area of interest centers on the regulation of protein transport across the membrane of the endoplasmic reticulum, a process that relies on the concerted action of three G proteins. We employ an integrative approach to these research areas combining structure determination with biochemical, biophysical and cell biological methods.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/sharp.html">Prof. Phillip Sharp</a>, </strong>E17-529, x3-6421, <a  href="mailto:sharppa@mit.edu">sharppa@mit.edu</a></dt>
  <dd><a href="http://web.mit.edu/sharplab/">Sharp Lab </a></dd>
  <dd>The ability to silence genes in mammalian cells through RNA interference (RNAi) has dramatically expanded the possibilities for genotype/phenotype analysis in cell biology. Investigation into the mechanisms responsible for the activities of short interfering RNAs (siRNAs) are ongoing with the objective of increasing their effectiveness in gene silencing. We are also investigating the roles of short RNAs in transcriptional silencing in murine embryonic stem cells. siRNAs have overlapping functions with microRNAs, endogenous genes in mammalian cells that, when paired by partial complementarity to an mRNA, inhibit accumulation of the corresponding protein. We are studying this translational repression and are also using RNAi technology to identify specific proteins important for the regulation of alternative RNA splicing and transcription.</dd>
  <dt><a href="http://mit.edu/biology/www/facultyareas/facresearch/sheng.html"><strong>Prof. Morgan Sheng</strong></a>, E18-215, x2-3716, <a href="mailto:msheng@mit.edu">msheng@mit.edu</a></dt>
  <dd><a href="mailto:msheng@mit.edu"></a>The Molecular Basis of Synaptic Plasticity. We seek to understand the molecular and cellular mechanisms underlying the ability of the brain to change in response to experience and to store information over long time periods, such as occur during development and for learning and memory. Our research is focused on the molecular regulation of synaptic structure and function, using genetic, biochemical, imaging and behavioral approaches in vitro and in vivo.</dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/sinskey.shtml">Prof. Anthony J. Sinskey</a>,</strong> 68-370, x3-6721, <a href="mailto:asinskey@mit.edu">asinskey@mit.edu</a>, <a href="http://mit.edu/org/b/biology/sinskey/www/home.html">Sinskey Lab </a></dt>
  <dd>The specific goals of our laboratory are to establish an interdisciplinary approach to metabolic engineering, focusing on the fundamental physiology, biochemistry and molecular genetics of important organisms. In particular, we are studying key factors that regulate the synthesis of different biomolecules. We apply metabolic engineering in several different project areas. Among prokaryotic systems, we study amino acid metabolism in Corynebacterium glutamicum, bioremediation and bioconversion processes in Rhodococcus, and biopolymer synthesis among Gram-negative bacteria such as Ralstonia eutropha. Among eukaryotic systems, we are studying lipid biosynthesis and embryogensis in oil palm and the accumulation of secondary metabolites in tropical plants.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/sive.shtml">Prof. Hazel Sive</a>,</strong>Whitehead 401C, x8-8242, <a  href="mailto:sive@wi.mit.edu">sive@wi.mit.edu</a>, <a href="http://web.wi.mit.edu/sive/pub/home.html">Sive Lab Home</a></dt>
  <dd> The questions of how an embryo decides where to place it's organs ("positional information") and how these organs are correctly organized into functional three dimensional structures ("morphogenesis") are of fundamental importance. We study these processes in the frog, Xenopus, and in the zebrafish, Danio. We have two major areas of interest: the nervous system, including very early patterning events as well as later events that build the three dimensional structure of the brain, and the extreme anterior of the embryo that forms the primary mouth, and is an evolutionarily conserved and important region. Frog and fish embryos are ideal for these studies, since the events we analyze take place very early in development, when mammalian embryos are tiny and inaccessible. Genes that are important for frog and fish embryogenesis are conserved in mammals, and our research is therefore relevant for understanding normal and abnormal human development.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/steiner.html">Prof. Lisa Steiner</a>,</strong> 68-271, x3-6704, <a  href="mailto:lsteiner@mit.edu">lsteiner@mit.edu</a></dt>
  <dd>The long-term interest of our lab is the evolution and development of the immune system. Our current goal is to understand the early development of cells in the lymphocytic lineage and of the organs in which these cells differentiate. Despite a wealth of information about later stages, little is known about early steps in the differentiation of B and T lymphocytes, including commitment to the lymphocytic lineages and homing of lymphocytic progenitors to the thymus. We are also interested in utilizing the emerging information about the zebrafish genome to describe the genetic loci encoding antigen-specific receptors and to analyze the expression and function of the genes within these loci.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/stubbe.shtml">Prof. Joanne Stubbe</a>, </strong>18-408B, x3-1814, <a  href="mailto:stubbe@mit.edu">stubbe@mit.edu</a></dt>
  <dd><a href="http://web.mit.edu/biochemistry/">Stubbe Lab </a></dd>
  <dd>General Research Interests: Role of Ribonucleotide Reductases in replication and repair; Iron homeostasis and metal cluster assembly in yeast; Mechanism of assembly of the metallo-cofactors required for nucleotide reduction using time resolved physical biochemical methods; Study of clinically active compounds that inactivate reductases and function as antitumor agents; Evolution and regulation of all the enzymes in the Purine Biosynthetic Pathway; Mechanism of DNA cleavers used clinically as antitumor agents; 2D NMR spectroscopy to determine structure of these drugs complexed with the DNA and the structure of the lesioned DNA; Mechanism of repair of the lesions; Mechanism in vivo and in vitro of polyester biosynthesis and homeostasis to generate biodegradable polymers with properites of thermoplastics; rubber biosynthesis.</dd>
  <dt><strong><a  href="http://mit.edu/biology/www/facultyareas/facresearch/tonegawa.shtml">Prof. Susumu Tonegawa</a>, </strong>46-5295, x3-6459, <a  href="mailto:tonegawa@mit.edu">tonegawa@mit.edu</a></dt>
  <dd>Our primary research interests are the molecular, cellular, and neuronal circuitry mechanisms underlying acquisition, consolidation, and retrieval of hippocampus-dependent memory in rodents. To study these problems, we produce conditionally engineered (i.e., spatially targeted and/or temporally regulated) mice and analyze these mice by multifaceted methods, including molecular and cellular biology, in vitro and in vivo electrophysiology, and behavioral studies. We attempt to identify deficits at each of the multiple levels of complexity in specific brain areas or cell types and to determine which deficits underlie a specific aspect or type of learning or memory in conditionally engineered mice. Conditionally engineered mice provide not only powerful tools for studying the fundamental mechanisms underlying cognition and behavior but also excellent small animal models for neurological and psychiatric diseases.</dd>
  <dt><strong><a href="http://web.mit.edu/biology/www/facultyareas/facresearch/vanderheiden.html">Matthew Vander Heiden</a>, </strong>76-561, x5-4471, <a href="mailto:mvh@mit.edu">mvh@mit.edu</a></dt>
  <dd>Cell proliferation requires the conversion of nutrients into biomass. One of the first differences noted between cancer cells and normal cells was a difference in metabolism. We hypothesize that this metabolic difference provides insight into in how proliferating cells, including cancer cells, convert nutrients into the chemical components needed to proliferate. My laboratory is interested in understanding the biochemical pathways cells use to meet these metabolic requirements of cell proliferation. In addition, we utilize mouse models of cancer to translate our biochemical understanding of cancer metabolism into better cancer therapies. </dd>
  <dt><strong><a  href="http://web.mit.edu/biology/www/Ar/99walker.html">Prof. Graham C. Walker</a>, </strong>68-633, x3-6716, <a  href="mailto:gwalker@mit.edu">gwalker@mit.edu</a></dt>
  <dd>Structure-function analyses of proteins involved in DNA repair, mutagenesis and cell cycle control; interactions of these proteins with components of the replicative DNA polymerase. Protein-protein interactions controlling the action of DNA polymerases that operate on damaged DNA in both prokaryotes and eukaryotes. Roles of rhizobial exopolysaccharides in nodule invasion. Analyses of bacterial functions required for the chronic infections that underlie Rhizobium symbiosis and Brucella pathogenesis.</dd>
  <dt><a  href="http://web.mit.edu/biology/www/Ar/99weinberg.html"><strong>Prof. Robert Weinberg</strong></a>, Whitehead 301, x8-5159, <a  href="mailto:weinberg@wi.mit.edu">weinberg@wi.mit.edu</a>, <a href="http://web.wi.mit.edu/weinberg/pub/">Weinberg Lab </a></dt>
  <dd>Research in our laboratory is increasingly focusing on three major areas: First, what molecular and biochemical mechanisms are responsible for triggering cell senescence? Second, how does the stroma of a tumor, such as a carcinoma, influence the biology of the tumor as a whole? Third, how do cancer cells within a primary tumor acquire the ability to invade and metastasize?</dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/wilson.shtml">Prof. Matthew A. Wilson</a>,</strong> E18-370, x3-2046, <a href="mailto:mwilson@ai.mit.edu">mwilson@ai.mit.edu</a>, <a href="http://web.mit.edu/wilsonlab/index.html">Wilson Lab </a></dt>
  <dd>How is experience represented and stored within the brain? A fundamental tenet of modern brain theory has been that information is coded in the coordinated activity of neuronal ensembles. Research in the Wilson laboratory focuses on the study of information representation across large populations of neurons in the mammalian nervous system, as well as on the mechanisms that underlie formation and maintenance of distributed memories in freely behaving animals. To study the basis of these processes, the lab employs a combination of molecular genetic, electrophysiological, pharmacological, behavioral, and computational approaches. Using techniques that allow the simultaneous activity of ensembles of hundreds of single neurons to be examined in freely behaving animals, the lab examines how memories of places and events are encoded across networks of cells within the hippocampus a region of the brain long implicated in the processes underlying learning and memory.</dd>
  <dt><strong> <a href="http://mit.edu/biology/www/facultyareas/facresearch/yaffe.shtml">Prof. Michael Yaffe</a>,</strong> E18-580, x-2442, <a href="mailto:myaffe@mit.edu">myaffe@mit.edu</a></dt>
  <dd>Regulation of protein-protein interactions and signal transduction pathways by protein and lipid phosphorylation. Structure and function of modular signaling domains. Design of bioinformatics tools for proteomic analysis. The goal of our research is to understand how protein phosphorylation controls progression through the cell cycle at the molecular level and how defects in phosphorylation bypass normal cell cycle checkpoints and lead to human cancer. We also study how protein and lipid phosphorylation controls the inflammatory response in phagocytic cells</dd>
  <dt><strong><a href="http://mit.edu/biology/www/facultyareas/facresearch/young.html">Prof. Richard Young</a>,</strong>Whitehead 501C, x8-5218, <a href="mailto:young@wi.mit.edu">young@wi.mit.edu</a>, </dt>
  <dd><a href="http://web.wi.mit.edu/young/">Young Lab</a>
    </dt>
  </dd>
  <dd>Transcriptional regulatory circuitry in development and disease. Our laboratory is mapping the transcriptional regulatory circuitry that controls cell state and differentiation in mice and humans. We use experimental and computational technologies to determine how transcriptional regulators and chromatin modifying enzymes control gene expression programs in embryonic stem cells and differentiated cells. </dd>
    </dl>
<p></p>
    <p></p>
  </div>
<!-- InstanceEndEditable --> 
<!--end content -->
  <!--begin spotlight -->
  <div id="newspot"><!-- InstanceBeginEditable name="newstext" -->
    <h4>UROP Contacts</h4>
    <h5>UROP Coordinator:</h5>
    <h6><a href="mailto:gmbrown@mit.edu">Prof. Gene M. Brown</a></h6>
    <h5>Department Head</h5>
    <h6><a href="mailto:ckaiser@mit.edu">Prof. Chris Kaiser</a></h6>
    <h5>UROP Payroll:</h5>
    <h6><a href="mailto:mmango@mit.edu">Mary Mango</a></h6>
<h2></h2>
  <!-- InstanceEndEditable --></div>
  <!--end spotlight -->
<!-- begin footer -->
<div id="footer"> <a href="http://mit.edu/"><img src="../../images/mituaap.gif" alt="Massachusetts Institute of Technology (MIT): UAAP" width="439" height="102" border="0" align="middle" /></a>Comments &amp; Questions to: <a href="../../basicinfo/comments.html">urop@mit.edu</a>

</div>
<!-- end footer -->
<!-- Begin MIT-use only web reporting counter -->
<img src="http://counter.mit.edu/tally" alt=""/>
<!-- End MIT-use only web reporting counter -->
</div>
</body>
<!-- InstanceEnd --></html>
